spec/fixtures/w3c_distributed_tracing.json

```diff
@@ -1,11 +1,3 @@
-//
-// from https://github.com/w3c/distributed-tracing/blob/main/test/test_data.json
-//
-// NOTE: This file is manually copied from the above link and
-//       there is currently NO automation keeping it in sync with the upstream version.
-//
-// Latest commit 98f210e on Sep 24, 2019
-//
```

Comment:
Moved the comment to the commit message.

<img width="814" alt="image" src="https://github.com/elastic/apm-agent-ruby/assets/16325797/db1bc24e-6b44-4d9b-aee8-301af4975224">

Reply:
Oh, wait. This information is lost when merging with squashing.

Any suggestions where to store this information instead?